### ISSUE TITLE
Settings - Social media

### ISSUE THREAD

**Issue by payeisa** [StatCan/StatcanPublicationUAT#159]
Can we please add an intro text on the Social media screen, below the heading (i.e. similar to "Manage subjects")?

English: Connect with Statistics Canada on social media.
French: Connectez avec Statistique Canada sur les médias sociaux.

@Rajat-Sen FYI - to resolve one of the accessibility issues.

**Comment by ForumG-SC** [comment]
Done

**Comment by payeisa** [comment]
The text looks good in English and French!
The font size should be the same as the one used on the "Settings - Manage subjects" screen.

**Comment by ForumG-SC** [comment]
Done

**Comment by payeisa** [comment]
Validated.
